parquet-column/src/main/java/org/apache/parquet/column/statistics/BinaryStatistics.java

```diff
@@ -46,8 +46,8 @@ public void mergeStatisticsMinMax(Statistics stats) {
 
   @Override
   public void setMinMaxFromBytes(byte[] minBytes, byte[] maxBytes) {
-    max = Binary.fromByteArray(maxBytes);
-    min = Binary.fromByteArray(minBytes);
+    max = Binary.fromReusedByteArray(maxBytes);
+    min = Binary.fromReusedByteArray(minBytes);
```

Comment:
oh, this is only used in the read path right? so the backing bytes should never change, but we use .fromReusedByteArray so that copy() would do the right thing if it were ever used. Is that correct?